### PR TITLE
Refactor Demo.py to reflect SDk calls instead of API calls

### PR DESCRIPTION
**Pull request by manojsalunke85**
Refactor Demo.py to reflect SDk calls instead of API calls

Signed-off-by: manoj <manoj.salunke@intel.com>

### CODE REVIEW

File: tests/Demo.py [hunk 1]
```diff
@@ -163,16 +230,14 @@ def local_main(config):
...
-        LOGGER.info("Received Response: %s , \n \n ", response)
+        LOGGER.info("Input Request cannot be processed %s", input_json_str)
 
     exit(0)
 
 
 # -----------------------------------------------------------------------------
 def parse_command_line(config, args):
-    LOGGER.info("***************** AVALON *****************")
+    LOGGER.info('************** TRUSTED COMPUTE FRAMEWORK (TCF) *************')
```
Comment (danintel) [inline tests/Demo.py]: This is a merge error. Do not change this line--keep it as `AVALON` (not `TCF`)

File: tests/Demo.py [hunk 2]
```diff
@@ -108,32 +139,68 @@ def local_main(config):
                         LOGGER.info("********** Worker details Updated with "
                                     "Worker ID*********\n%s\n",
                                     input_json_str1)
+                        # Retrieve worker details
+                        LOGGER.info("****Request Json**** \n%s\n",
```
Comment (danintel) [inline tests/Demo.py]: Change `Json` to `JSON`

File: tests/Demo.py [hunk 3]
```diff
@@ -108,32 +139,68 @@ def local_main(config):
...
-            LOGGER.info("*********Request Json********* \n%s\n",
-                        input_json_str1)
-            response = uri_client._postmsg(input_json_str1)
-            LOGGER.info("**********Received Response*********\n%s\n", response)
-
+                input_json_obj = json.loads(input_json_str1)
+                jrpc_req_id = input_json_obj['id']
+                jrpc_req_id += 1
+                wo_id = input_json_obj["params"]["workOrderId"]
+                LOGGER.info("*********Request Json********* \n%s\n",
```
Comment (danintel) [inline tests/Demo.py]: Change `Json` to `JSON`

File: tests/Demo.py [hunk 4]
```diff
@@ -108,32 +139,68 @@ def local_main(config):
...
+                work_order = JRPCWorkOrderImpl(config)
+                response = work_order.work_order_get_result(wo_id, jrpc_req_id)
+                LOGGER.info("*******Received Response*******: %s, \n \n ",
+                            response)
             # -----------------------------------------------------------------
+            if "WorkerLookUp" in input_json_str1:
+                input_json_obj = json.loads(input_json_str1)
+                # Prepare worker
+                # Read JRPC ID from JSON
+                LOGGER.info("********Request Json********* \n%s\n",
```
Comment (danintel) [inline tests/Demo.py]: Change `Json` to `JSON`

**Comment by Ram-srini** [comment]
@manojsalunke85 Please do rebase, it has some merge conflict.

**Comment by rranjan3** [comment]
Demo.py planned to be removed altogether. Closing this PR for it.

**Pull request closed without merging**